### PR TITLE
New Version without Depreciation Errors

### PR DESCRIPTION
**Pull request by cwhittl**

### CODE REVIEW

**Comment by bobisjan** [comment]
Hi 👋,

is this ready for review? I would love to see this merged and released 😍.

/cc @scalvert @nickiaconis

File: tests/dummy/app/templates/application.hbs [hunk 1]
```diff
@@ -1,17 +1,3 @@
...
-    {{#tm.target}}
-      supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious
-    {{/tm.target}}
-    {{#tm.button}}
-      <span aria-hidden="true">{{if tm.isTruncated "⬇️" "⬆️"}}</span>
-      <span class="truncate-multiline--visually-hidden">{{if tm.isTruncated "view more text" "view less text"}}</span>
-    {{/tm.button}}
-  {{/truncate-multiline}}
-</p>
+{{outlet}}
```
Comment (scalvert) [inline tests/dummy/app/templates/application.hbs]: I think you probably upgraded with `ember init`, but we should probably put this back.

**Comment by cwhittl** [comment]
@scalvert this has been remedied

**Comment by mhluska** [comment]
@scalvert do you need help? Is it a matter of merging release into master? I can probably look into it.

**Comment by scalvert** [comment]
Yes. Basically the branching structure is a bit different than what I'm used to. Master seems to not be latest, and there are persistent release branches. I'd prefer to merge latest into master, and release from there.

Overall, this repo needs a little bit of love but I just hasn't had the time to prioritize. I'd be happy for the help.

**Comment by mhluska** [comment]
@scalvert I pulled both the release-0.3 and release-1.0 branches into master:

0.3: https://github.com/ember-truncate/ember-truncate/pull/58
1.0: https://github.com/ember-truncate/ember-truncate/pull/59

I think we should merge the 0.3, then update the 1.0 and merge that one. We could also [add a package-lock file after](https://github.com/ember-truncate/ember-truncate/issues/57)

**Pull request merged**